client/src/styles/global.scss

```diff
@@ -44,13 +39,13 @@ body {
 a {
     text-decoration: none;
     color: $brand-pink;
-    transition: .2s ease-out;
+    transition: 0.2s ease-out;
 
     &:hover,
     &:focus {
         color: darken($brand-pink, 15%);
         text-decoration: none;
-        transform: translate3d(0, -.1rem, 0);
+        transform: translate3d(0, -0.1rem, 0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)